truss-chains/truss_chains/deploy.py

```diff
@@ -221,4 +250,25 @@ def deploy_remotely(
             chainlet_name_to_url[chainlet_descriptor.name] = service.predict_url
         else:
             chainlet_name_to_url[chainlet_descriptor.name] = "http://dummy"
+
+    if isinstance(options, definitions.DeploymentOptionsBaseten):
+        chainlets: List[ChainletData] = []
+        entrypoint_name = chain_service.entrypoint_name
+
+        for chainlet_name, truss_service in chain_service.services.items():
+            baseten_service = cast(b10_service.BasetenService, truss_service)
+            chainlets.append(
+                ChainletData(
+                    name=chainlet_name,
+                    oracle_version_id=baseten_service.model_version_id,
+                    is_entrypoint=chainlet_name == entrypoint_name,
```

Comment:
You could also do `chain_service.get_entrypoint == baseten_service` and remove the `entrypoint_name` from the API.

Also thinking, since `get_entrypoint` is a property, it should not have `get` in the name, or made a method. And maybe we should rename the argument/variable `entrypoint` (where it's a string) to `entrypoint_name` - this is not super clean right now.